Output/UARTOut/arm/uart_serial.c

```diff
@@ -172,6 +204,10 @@ void uart_serial_setup()
 	PORTD_PCR2 = PORT_PCR_PE | PORT_PCR_PS | PORT_PCR_PFE | PORT_PCR_MUX(3); // RX Pin
 	PORTD_PCR3 = PORT_PCR_DSE | PORT_PCR_SRE | PORT_PCR_MUX(3); // TX Pin
 
+// ???
+#elif defined(_kii_v3_)
```

Comment:
This is the GPIO pin configuration. i.e. telling the MUX that the specified GPIO pins should be for the UART.

PA21 (D_RX1)  and PA22 (D_TX1).

Reply:
Is the muxing relevant to all arm cortex devices, or just the kinetis line. Skimming the sam doc I haven't been able to find anything describing such a feature.